graphql/resolvers/exerciseSubmissionCrud.ts

```diff
@@ -0,0 +1,46 @@
+import { MutationAddExerciseSubmissionArgs } from '..'
+import { Context } from '../../@types/helpers'
+import prisma from '../../prisma'
+
+export const exerciseSubmissions = (
+  _parent: void,
+  _args: void,
+  context: Context
+) => {
+  const userId = context.req.user?.id
+  if (!userId) return []
```

Comment:
I decided to return an empty array here, but maybe I should throw instead since you need to be logged in to get your own exercise submissions.

Reply:
Returning an empty array make it more pleasant because we won't have to wrap it with try/catch block.

I'd have to look into the benefits of throwing errors in graphql resolvers in order to decide.